### PR TITLE
Fixing LGTM size_t

### PR DESCRIPTION
**Pull request by jandom**
##### Description

- use size_t over unsigned int 
- avoids integer overflow on multiplication

##### Target release

No preference

##### Type of contribution

- [x] changes to code or doc authored by PLUMED developers
- [ ] changes to a module not authored by you
- [ ] new module contribution or edit of a module authored by you

##### Copyright

- [x] I agree to transfer the copyright of the code I have written to the PLUMED developers or to the author of the code I am modifying.

- [ ] the module I added or modified contains a `COPYRIGHT` file with the correct license information. Code should be released under an open source license. I also used the command `cd src && ./header.sh mymodulename` in order to make sure the headers of the module are correct. 

##### Tests

- [x] I added a new regtest or modified an existing regtest to validate my changes.
- [x] I verified that all regtests are passed successfully on Travis-CI.

### CODE REVIEW

**Comment by jandom** [comment]
There appear to be some very small coverage changes – I'm not sure how harmonizing the size_t issue would have caused that? (no lines of code were added/removed)

**Comment by carlocamilloni** [comment]
no don't worry that is just noise, the code coverage is not 100% reproducible between runs

**Comment by jandom** [comment]
Oh, that's great, thanks @carlocamilloni – I'll push a few more of these fixes here, removing all the size_t and some time next week this should be wrapped up

**Comment by jandom** [comment]
Guys, I think this is as far as I'm going to push it – just some type shuffling. 

Haven't fully eliminated the problem – to do that would take a much larger refactor, that I'm not even sure I know how to do.

**Comment by jandom** [comment]
Ready for review?

File: .travis.yml [hunk 1]
```diff
@@ -116,6 +116,7 @@ install:
       brew update > /dev/null ;
       brew install ccache ;
       brew install gawk ;
+      brew upgrade gnupg ;
```
Comment (GiovanniBussi) [inline .travis.yml:119]: @jandom can you explain this change?

Comment (jandom) [reply .travis.yml:119]: @GiovanniBussi good catch! that was just to try the MacOS build to work – it complains about a gnupg version being missing (while another version for the package is present). I was just shooting in the dark here, clearly didn't make a difference

This has been removed now

Comment (jandom) [reply .travis.yml]: Sorry misspoke, now its removed

**Comment by GiovanniBussi** [comment]
This is ok with me. I cc also authors of other modules that you fixed in case they wish to comments. If I don't hear anything in a couple of days, I will merge to master. Thanks a lot @jandom !

cc: @carlocamilloni @maxbonomi @gtribello @SilvioPipolo @valsson

**Comment by valsson** [comment]
I am fine with the changes in `src/ves/MD_LinearExpansionPES.cpp`

**Comment by SilvioPipolo** [comment]
Same for me I'm fine with changes in PIV.cpp. Thank you.
Best,
Silvio

**Comment by maxbonomi** [comment]
Same here!!!

**Comment by jandom** [comment]
Thanks everyone, it's a really small and silly PR but I'm happy it made it in

**Pull request merged**